bugbot/rules/webcompat_sightline.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatSightline(BzCleaner):
+    normal_changes_max = 1000
```

Comment:
This limit is too high. We have this mechanism as a safety net. The number should be equal to the maximum number of bugs that this rule would modify in a regular cron run (i.e., hourly in this case).

If you drop the attribute altogether, the result would be 50 bugs.

To modify old bugs, you do not need this attribute; I could manually run it in the no-limit mode for one time.

Reply:
OK, as long as it's only counting bugs that actually require an update, I guess that's fine.